CTP/sql/bin/run.sh

```diff
@@ -1,4 +1,5 @@
 #!/bin/bash
+set -x
```

Comment:
1. Please add a space between '#' sign and comment for each line of the license stmt.
2. Is it still need the "set -x" ?